examples/bri-3/src/bri/workgroup/workgroups/api/workgroups.controller.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  Param,
+  Post,
+  Put,
+} from '@nestjs/common';
+import { CommandBus, QueryBus } from '@nestjs/cqrs';
+import { CreateWorkgroupCommand } from '../capabilities/createWorkgroup/createWorkgroup.command';
+import { DeleteWorkgroupCommand } from '../capabilities/deleteWorkgroup/deleteWorkgroup.command';
+import { GetWorkgroupByIdQuery } from '../capabilities/getWorkgroupById/getWorkgroupById.query';
+import { UpdateWorkgroupCommand } from '../capabilities/updateWorkgroup/updateWorkgroup.command';
+import { CreateWorkgroupDto } from './dtos/request/createWorkgroup.dto';
+import { UpdateWorkgroupDto } from './dtos/request/updateWorkgroup.dto';
+import { WorkgroupDto } from './dtos/response/workgroup.dto';
+
+@Controller('workgroups')
+export class WorkgroupController {
+  constructor(private commandBus: CommandBus, private queryBus: QueryBus) {}
+
+  @Get('/:id')
+  async getworkgroupById(@Param('id') id: string): Promise<WorkgroupDto> {
+    return await this.queryBus.execute(new GetWorkgroupByIdQuery(id));
+  }
+
+  @Post()
+  async CreateWorkgroup(
+    @Body() requestDto: CreateWorkgroupDto,
+  ): Promise<string> {
+    return await this.commandBus.execute(
+      new CreateWorkgroupCommand(
+        requestDto.name,
+        requestDto.administratorIds,
+        requestDto.securityPolicy,
+        requestDto.privacyPolicy,
+        requestDto.participantIds,
+        requestDto.workstepIds,
+        requestDto.workflowIds,
+      ),
+    );
+  }
+
+  @Put('/:id')
+  async updateWorkgroup(
+    @Param('id') id: string,
+    @Body() requestDto: UpdateWorkgroupDto,
+  ): Promise<void> {
+    return await this.commandBus.execute(
+      new UpdateWorkgroupCommand(
+        id,
+        requestDto.name,
+        requestDto.administratorIds,
+        requestDto.securityPolicy,
+        requestDto.privacyPolicy,
+        requestDto.participantIds,
+      ),
+    );
+  }
+
+  @Delete('/:id')
+  async DeleteWorkgroupCommand(@Param('id') id: string): Promise<void> {
```

Comment:
deleteWorkgroup instead of DeleteWorkgroupCommand

Reply:
renamed